### ISSUE TITLE
Unable to start server 

### ISSUE THREAD

**Issue by SmuleChez** [TunkDesign/NeMa#61]
When I try to start up my server, i get the same error every time, and reinstalling NeMa does not seems to fix the issue.

Starting MineCraft Server..
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/TunkDesign/Build/NeMa/build/NeMa/out00-PYZ.pyz/threading", line 810, in __bootstrap_inner
  File "/TunkDesign/Build/NeMa/build/NeMa/out00-PYZ.pyz/threading", line 763, in run
  File "/TunkDesign/Build/NeMa/build/NeMa/out00-PYZ.pyz/scripts.process", line 760, in console
ValueError: unsupported format character '!' (0x21) at index 19

MineCraft Server was killed outside of NeManager!

**Comment by TunkDesign** [comment]
Hey @SmuleChez!

Could you please post the contents of the Nema.log file?
Also, what operating system are you using?

Thanks :)

**Comment by SmuleChez** [comment]
Hello! I am using Ubuntu server 19.04 for my OS, here is what is in the log file.
Thank you for the help!

[1615 | INFO | 20/06/2019 07:05:22 PM] *******************************
[1612 | INFO | 20/06/2019 07:05:22 PM] * New Wrapper Session Started *
[1612 | INFO | 20/06/2019 07:05:22 PM] *******************************
[1612 | INFO | 20/06/2019 07:05:22 PM] Checking if we should fix something before starting the wrapper..
[1602 | WARNING | 20/06/2019 07:05:22 PM] Configuration not found. Creating..
[1612 | INFO | 20/06/2019 07:05:22 PM] Starting config creations..
[1612 | INFO | 20/06/2019 07:05:22 PM] Loading defaults..
[1602 | WARNING | 20/06/2019 07:05:22 PM] Database not found. Creating..
[1612 | INFO | 20/06/2019 07:05:22 PM] Checking if data directory and database file exists..
[1602 | WARNING | 20/06/2019 07:05:22 PM] Data folder doesn't exist. Creating everything needed to continue..
[1612 | INFO | 20/06/2019 07:05:22 PM] Connecting to database..
[1612 | INFO | 20/06/2019 07:05:22 PM] Creating server table..
[1612 | INFO | 20/06/2019 07:05:22 PM] Creating schedule table..
[1612 | INFO | 20/06/2019 07:05:22 PM] Creating ftp table..
[1612 | INFO | 20/06/2019 07:05:22 PM] Creating cloud backup table..
[1612 | INFO | 20/06/2019 07:05:22 PM] Database setup done!
[1602 | WARNING | 20/06/2019 07:05:22 PM] Wrapper key not found. Generating a new key..
[1612 | INFO | 20/06/2019 07:05:22 PM] ----------
[1612 | INFO | 20/06/2019 07:05:22 PM] Generating wrapper key..
[1612 | INFO | 20/06/2019 07:05:22 PM] Key generated!
[1612 | INFO | 20/06/2019 07:05:22 PM] Writing encryped key to nema.key..
[1612 | INFO | 20/06/2019 07:05:22 PM] Showing key in wrapper console.
[1602 | WARNING | 20/06/2019 07:05:22 PM] Jar directory not found. Creating..
[1602 | WARNING | 20/06/2019 07:05:22 PM] Server directory not found. Creating..
[1602 | WARNING | 20/06/2019 07:05:22 PM] Backup directory not found. Creating..
[1612 | INFO | 20/06/2019 07:05:22 PM] ----------
[1612 | INFO | 20/06/2019 07:05:23 PM] Initiating schedule worker..
[1612 | INFO | 20/06/2019 07:05:23 PM] Initiating file worker..
[1612 | INFO | 20/06/2019 07:05:23 PM] Initiating wrapper startup..
[1612 | INFO | 20/06/2019 07:05:23 PM] Binding socket to port 20000
[1620 | DEBUG | 20/06/2019 07:05:23 PM] Setting address to be reusable
[1612 | INFO | 20/06/2019 07:05:23 PM] Socket listening.
[1612 | INFO | 20/06/2019 07:08:33 PM] *******************************
[1612 | INFO | 20/06/2019 07:08:33 PM] * New Wrapper Session Started *
[1612 | INFO | 20/06/2019 07:08:33 PM] *******************************
[1612 | INFO | 20/06/2019 07:08:34 PM] Checking if we should fix something before starting the wrapper..
[1612 | INFO | 20/06/2019 07:08:34 PM] Found configuration!
[1612 | INFO | 20/06/2019 07:08:34 PM] Found database!
[1612 | INFO | 20/06/2019 07:08:34 PM] Found wrapper key!
[1612 | INFO | 20/06/2019 07:08:34 PM] Trying to load key..
[1612 | INFO | 20/06/2019 07:08:34 PM] Key loaded into system!
[1612 | INFO | 20/06/2019 07:08:34 PM] Found jar directory!
[1612 | INFO | 20/06/2019 07:08:34 PM] Found server directory!
[1612 | INFO | 20/06/2019 07:08:34 PM] Found backup directory!
[1612 | INFO | 20/06/2019 07:08:34 PM] ----------
[1612 | INFO | 20/06/2019 07:08:34 PM] Initiating schedule worker..
[1612 | INFO | 20/06/2019 07:08:34 PM] Initiating file worker..
[1612 | INFO | 20/06/2019 07:08:34 PM] Initiating wrapper startup..
[1612 | INFO | 20/06/2019 07:08:34 PM] Binding socket to port 20000
[1620 | DEBUG | 20/06/2019 07:08:34 PM] Setting address to be reusable
[1612 | INFO | 20/06/2019 07:08:34 PM] Socket listening.
[1612 | INFO | 20/06/2019 08:24:06 PM] *******************************
[1612 | INFO | 20/06/2019 08:24:06 PM] * New Wrapper Session Started *
[1612 | INFO | 20/06/2019 08:24:06 PM] *******************************
[1612 | INFO | 20/06/2019 08:24:06 PM] Checking if we should fix something before starting the wrapper..
[1612 | INFO | 20/06/2019 08:24:06 PM] Found configuration!
[1612 | INFO | 20/06/2019 08:24:06 PM] Found database!
[1612 | INFO | 20/06/2019 08:24:06 PM] Found wrapper key!
[1612 | INFO | 20/06/2019 08:24:06 PM] Trying to load key..
[1612 | INFO | 20/06/2019 08:24:06 PM] Key loaded into system!
[1612 | INFO | 20/06/2019 08:24:06 PM] Found jar directory!
[1612 | INFO | 20/06/2019 08:24:06 PM] Found server directory!
[1612 | INFO | 20/06/2019 08:24:06 PM] Found backup directory!
[1612 | INFO | 20/06/2019 08:24:06 PM] ----------
[1612 | INFO | 20/06/2019 08:24:06 PM] Initiating schedule worker..
[1612 | INFO | 20/06/2019 08:24:06 PM] Initiating file worker..
[1612 | INFO | 20/06/2019 08:24:06 PM] Initiating wrapper startup..
[1612 | INFO | 20/06/2019 08:24:06 PM] Binding socket to port 20000
[1620 | DEBUG | 20/06/2019 08:24:06 PM] Setting address to be reusable
[1612 | INFO | 20/06/2019 08:24:06 PM] Socket listening.
[1612 | INFO | 20/06/2019 08:24:57 PM] ----------
[1612 | INFO | 20/06/2019 08:24:57 PM] Starting creation of new server..
[1612 | INFO | 20/06/2019 08:24:57 PM] Creating a new server folder..
[1612 | INFO | 20/06/2019 08:24:57 PM] Creating database entry for this server..
[130 | DEBUG | 20/06/2019 08:24:57 PM] [debug] closing IOLoop (<pyftpdlib.ioloop.Epoll (fds=1, tasks=0) at 0xa3223ec>)
[1620 | DEBUG | 20/06/2019 08:24:57 PM] Connecting to database..
[1620 | DEBUG | 20/06/2019 08:24:57 PM] Inserting data..
[1620 | DEBUG | 20/06/2019 08:24:57 PM] Connecting to database..
[1620 | DEBUG | 20/06/2019 08:24:57 PM] Inserting data..
[1612 | INFO | 20/06/2019 08:24:57 PM] Creating jar file..
[1612 | INFO | 20/06/2019 08:24:57 PM] Starting server properties generation..
[1612 | INFO | 20/06/2019 08:31:06 PM] ----------
[1612 | INFO | 20/06/2019 08:31:06 PM] Loading server startup sequence..
[1612 | INFO | 20/06/2019 08:31:06 PM] Resetting server status..
[1612 | INFO | 20/06/2019 08:31:06 PM] Fetching server data..
[1612 | INFO | 20/06/2019 08:31:06 PM] Data fetched! Trying to start MineCraft Server..
[1612 | INFO | 20/06/2019 08:31:06 PM] Configuring wrapper for start of MineCraft Server..
[1612 | INFO | 20/06/2019 08:31:06 PM] Checking OS type..
[1612 | INFO | 20/06/2019 08:31:06 PM] Compiling startup command..
[1612 | INFO | 20/06/2019 08:31:06 PM] Compiled command: java -client -Duser.language=en -Xms1024M -Xmx16384M  -jar server.jar nogui --nojline
[1612 | INFO | 20/06/2019 08:31:06 PM] Continuing startup of MineCraft Server..
[1612 | INFO | 20/06/2019 08:31:06 PM] Updating global serverlist with server data..
[1620 | DEBUG | 20/06/2019 08:31:06 PM] Serverlist updated with {22b4c40b9275d91a8c3a38e139207ab4, <subprocess.Popen object at 0xa3245ec>, 9345}
[1612 | INFO | 20/06/2019 08:31:06 PM] Starting server daemon..
[1612 | INFO | 20/06/2019 08:31:06 PM] Server daemon started!
[1612 | INFO | 20/06/2019 08:31:06 PM] Set global server status to 'starting'..
[1612 | INFO | 20/06/2019 08:31:06 PM] MineCraft Server was successfully started!
[1612 | INFO | 20/06/2019 08:31:16 PM] ----------
[1612 | INFO | 20/06/2019 08:31:16 PM] Loading server startup sequence..
[1612 | INFO | 20/06/2019 08:31:16 PM] Resetting server status..
[1612 | INFO | 20/06/2019 08:31:16 PM] Fetching server data..
[1612 | INFO | 20/06/2019 08:31:16 PM] Data fetched! Trying to start MineCraft Server..
[1586 | ERROR | 20/06/2019 08:31:16 PM] Failed to start server. This server is already running.
[1612 | INFO | 20/06/2019 08:31:59 PM] ----------
[1612 | INFO | 20/06/2019 08:31:59 PM] Loading server force stop sequence..
[1612 | INFO | 20/06/2019 08:31:59 PM] Fetching server data..
[1612 | INFO | 20/06/2019 08:31:59 PM] Data fetched! Trying to force stop MineCraft Server..
[1612 | INFO | 20/06/2019 08:32:38 PM] ----------
[1612 | INFO | 20/06/2019 08:32:38 PM] Loading server force stop sequence..
[1612 | INFO | 20/06/2019 08:32:38 PM] Fetching server data..
[1612 | INFO | 20/06/2019 08:32:38 PM] Data fetched! Trying to force stop MineCraft Server..
[1586 | ERROR | 20/06/2019 08:35:14 PM] Oops, something went wrong! Failed to inject command.
Traceback (most recent call last):
  File "/TunkDesign/Build/NeMa/build/NeMa/out00-PYZ.pyz/scripts.process", line 634, in cmd
IOError: [Errno 32] Broken pipe
[1586 | ERROR | 20/06/2019 08:35:17 PM] Oops, something went wrong! Failed to inject command.
Traceback (most recent call last):
  File "/TunkDesign/Build/NeMa/build/NeMa/out00-PYZ.pyz/scripts.process", line 634, in cmd
IOError: [Errno 32] Broken pipe
[1612 | INFO | 20/06/2019 08:35:30 PM] ----------
[1612 | INFO | 20/06/2019 08:35:30 PM] Loading server force stop sequence..
[1612 | INFO | 20/06/2019 08:35:30 PM] Fetching server data..
[1612 | INFO | 20/06/2019 08:35:30 PM] Data fetched! Trying to force stop MineCraft Server..
[1612 | INFO | 20/06/2019 09:08:36 PM] *******************************
[1612 | INFO | 20/06/2019 09:08:36 PM] * New Wrapper Session Started *
[1612 | INFO | 20/06/2019 09:08:36 PM] *******************************
[1612 | INFO | 20/06/2019 09:08:37 PM] Checking if we should fix something before starting the wrapper..
[1612 | INFO | 20/06/2019 09:08:37 PM] Found configuration!
[1612 | INFO | 20/06/2019 09:08:37 PM] Found database!
[1612 | INFO | 20/06/2019 09:08:37 PM] Found wrapper key!
[1612 | INFO | 20/06/2019 09:08:37 PM] Trying to load key..
[1612 | INFO | 20/06/2019 09:08:37 PM] Key loaded into system!
[1612 | INFO | 20/06/2019 09:08:37 PM] Found jar directory!
[1612 | INFO | 20/06/2019 09:08:37 PM] Found server directory!
[1602 | WARNING | 20/06/2019 09:08:37 PM] Backup directory not found. Creating..
[1612 | INFO | 20/06/2019 09:08:37 PM] ----------
[1612 | INFO | 20/06/2019 09:08:37 PM] Initiating schedule worker..
[1612 | INFO | 20/06/2019 09:08:37 PM] Initiating file worker..
[1612 | INFO | 20/06/2019 09:08:37 PM] Initiating wrapper startup..
[1612 | INFO | 20/06/2019 09:08:37 PM] Binding socket to port 20000
[1620 | DEBUG | 20/06/2019 09:08:37 PM] Setting address to be reusable
[1612 | INFO | 20/06/2019 09:08:37 PM] Socket listening.
[1612 | INFO | 20/06/2019 09:11:38 PM] *******************************
[1612 | INFO | 20/06/2019 09:11:38 PM] * New Wrapper Session Started *
[1612 | INFO | 20/06/2019 09:11:38 PM] *******************************
[1612 | INFO | 20/06/2019 09:11:38 PM] Checking if we should fix something before starting the wrapper..
[1612 | INFO | 20/06/2019 09:11:38 PM] Found configuration!
[1612 | INFO | 20/06/2019 09:11:38 PM] Found database!
[1612 | INFO | 20/06/2019 09:11:38 PM] Found wrapper key!
[1612 | INFO | 20/06/2019 09:11:38 PM] Trying to load key..
[1612 | INFO | 20/06/2019 09:11:38 PM] Key loaded into system!
[1612 | INFO | 20/06/2019 09:11:38 PM] Found jar directory!
[1612 | INFO | 20/06/2019 09:11:38 PM] Found server directory!
[1612 | INFO | 20/06/2019 09:11:38 PM] Found backup directory!
[1612 | INFO | 20/06/2019 09:11:38 PM] ----------
[1612 | INFO | 20/06/2019 09:11:38 PM] Initiating schedule worker..
[1612 | INFO | 20/06/2019 09:11:38 PM] Initiating file worker..
[1612 | INFO | 20/06/2019 09:11:38 PM] Initiating wrapper startup..
[1612 | INFO | 20/06/2019 09:11:38 PM] Binding socket to port 20000
[1620 | DEBUG | 20/06/2019 09:11:38 PM] Setting address to be reusable
[1612 | INFO | 20/06/2019 09:11:38 PM] Socket listening.
[1612 | INFO | 20/06/2019 09:15:00 PM] ----------
[1612 | INFO | 20/06/2019 09:15:00 PM] Loading server startup sequence..
[1612 | INFO | 20/06/2019 09:15:00 PM] Resetting server status..
[1612 | INFO | 20/06/2019 09:15:00 PM] Fetching server data..
[1612 | INFO | 20/06/2019 09:15:00 PM] Data fetched! Trying to start MineCraft Server..
[1612 | INFO | 20/06/2019 09:15:00 PM] Configuring wrapper for start of MineCraft Server..
[1612 | INFO | 20/06/2019 09:15:00 PM] Checking OS type..
[1612 | INFO | 20/06/2019 09:15:00 PM] Compiling startup command..
[1612 | INFO | 20/06/2019 09:15:00 PM] Compiled command: java -client -Duser.language=en -Xms1024M -Xmx16384M  -jar server.jar nogui --nojline
[1612 | INFO | 20/06/2019 09:15:00 PM] Continuing startup of MineCraft Server..
[1612 | INFO | 20/06/2019 09:15:00 PM] Updating global serverlist with server data..
[1620 | DEBUG | 20/06/2019 09:15:00 PM] Serverlist updated with {22b4c40b9275d91a8c3a38e139207ab4, <subprocess.Popen object at 0x997d94c>, 10097}
[1612 | INFO | 20/06/2019 09:15:00 PM] Starting server daemon..
[1612 | INFO | 20/06/2019 09:15:00 PM] Server daemon started!
[1612 | INFO | 20/06/2019 09:15:00 PM] Set global server status to 'starting'..
[1612 | INFO | 20/06/2019 09:15:00 PM] MineCraft Server was successfully started!
[1612 | INFO | 20/06/2019 09:15:41 PM] ----------
[1612 | INFO | 20/06/2019 09:15:41 PM] Loading server force stop sequence..
[1612 | INFO | 20/06/2019 09:15:41 PM] Fetching server data..
[1612 | INFO | 20/06/2019 09:15:41 PM] Data fetched! Trying to force stop MineCraft Server..
[1612 | INFO | 20/06/2019 09:17:03 PM] *******************************
[1612 | INFO | 20/06/2019 09:17:03 PM] * New Wrapper Session Started *
[1612 | INFO | 20/06/2019 09:17:03 PM] *******************************
[1612 | INFO | 20/06/2019 09:17:04 PM] Checking if we should fix something before starting the wrapper..
[1612 | INFO | 20/06/2019 09:17:04 PM] Found configuration!
[1612 | INFO | 20/06/2019 09:17:04 PM] Found database!
[1612 | INFO | 20/06/2019 09:17:04 PM] Found wrapper key!
[1612 | INFO | 20/06/2019 09:17:04 PM] Trying to load key..
[1612 | INFO | 20/06/2019 09:17:04 PM] Key loaded into system!
[1612 | INFO | 20/06/2019 09:17:04 PM] Found jar directory!
[1612 | INFO | 20/06/2019 09:17:04 PM] Found server directory!
[1612 | INFO | 20/06/2019 09:17:04 PM] Found backup directory!
[1612 | INFO | 20/06/2019 09:17:04 PM] ----------
[1612 | INFO | 20/06/2019 09:17:04 PM] Initiating schedule worker..
[1612 | INFO | 20/06/2019 09:17:04 PM] Initiating file worker..
[1612 | INFO | 20/06/2019 09:17:04 PM] Initiating wrapper startup..
[1612 | INFO | 20/06/2019 09:17:04 PM] Binding socket to port 20000
[1620 | DEBUG | 20/06/2019 09:17:04 PM] Setting address to be reusable
[1612 | INFO | 20/06/2019 09:17:04 PM] Socket listening.
[1612 | INFO | 20/06/2019 09:18:21 PM] ----------
[1612 | INFO | 20/06/2019 09:18:21 PM] Loading server startup sequence..
[1612 | INFO | 20/06/2019 09:18:21 PM] Resetting server status..
[1612 | INFO | 20/06/2019 09:18:21 PM] Fetching server data..
[1612 | INFO | 20/06/2019 09:18:21 PM] Data fetched! Trying to start MineCraft Server..
[1612 | INFO | 20/06/2019 09:18:21 PM] Configuring wrapper for start of MineCraft Server..
[1612 | INFO | 20/06/2019 09:18:21 PM] Checking OS type..
[1612 | INFO | 20/06/2019 09:18:21 PM] Compiling startup command..
[1612 | INFO | 20/06/2019 09:18:21 PM] Compiled command: java -client -Duser.language=en -Xms1024M -Xmx16384M  -jar server.jar nogui --nojline
[1612 | INFO | 20/06/2019 09:18:21 PM] Continuing startup of MineCraft Server..
[1612 | INFO | 20/06/2019 09:18:21 PM] Updating global serverlist with server data..
[1620 | DEBUG | 20/06/2019 09:18:21 PM] Serverlist updated with {22b4c40b9275d91a8c3a38e139207ab4, <subprocess.Popen object at 0x887312c>, 10155}
[1612 | INFO | 20/06/2019 09:18:21 PM] Starting server daemon..
[1612 | INFO | 20/06/2019 09:18:21 PM] Server daemon started!
[1612 | INFO | 20/06/2019 09:18:21 PM] Set global server status to 'starting'..
[1612 | INFO | 20/06/2019 09:18:21 PM] MineCraft Server was successfully started!
[1612 | INFO | 20/06/2019 09:18:29 PM] ----------
[1612 | INFO | 20/06/2019 09:18:29 PM] Loading server startup sequence..
[1612 | INFO | 20/06/2019 09:18:29 PM] Resetting server status..
[1612 | INFO | 20/06/2019 09:18:29 PM] Fetching server data..
[1612 | INFO | 20/06/2019 09:18:29 PM] Data fetched! Trying to start MineCraft Server..
[1612 | INFO | 20/06/2019 09:18:29 PM] Configuring wrapper for start of MineCraft Server..
[1612 | INFO | 20/06/2019 09:18:29 PM] Checking OS type..
[1612 | INFO | 20/06/2019 09:18:29 PM] Compiling startup command..
[1612 | INFO | 20/06/2019 09:18:29 PM] Compiled command: java -client -Duser.language=en -Xms1024M -Xmx16384M  -jar server.jar nogui --nojline
[1612 | INFO | 20/06/2019 09:18:29 PM] Continuing startup of MineCraft Server..
[1612 | INFO | 20/06/2019 09:18:29 PM] Updating global serverlist with server data..
[1620 | DEBUG | 20/06/2019 09:18:29 PM] Serverlist updated with {22b4c40b9275d91a8c3a38e139207ab4, <subprocess.Popen object at 0x8873dcc>, 10159}
[1612 | INFO | 20/06/2019 09:18:29 PM] Starting server daemon..
[1612 | INFO | 20/06/2019 09:18:29 PM] Server daemon started!
[1612 | INFO | 20/06/2019 09:18:29 PM] Set global server status to 'starting'..
[1612 | INFO | 20/06/2019 09:18:29 PM] MineCraft Server was successfully started!
[1612 | INFO | 20/06/2019 09:19:25 PM] *******************************
[1612 | INFO | 20/06/2019 09:19:25 PM] * New Wrapper Session Started *
[1612 | INFO | 20/06/2019 09:19:25 PM] *******************************
[1612 | INFO | 20/06/2019 09:19:25 PM] Checking if we should fix something before starting the wrapper..
[1612 | INFO | 20/06/2019 09:19:25 PM] Found configuration!
[1612 | INFO | 20/06/2019 09:19:25 PM] Found database!
[1612 | INFO | 20/06/2019 09:19:25 PM] Found wrapper key!
[1612 | INFO | 20/06/2019 09:19:25 PM] Trying to load key..
[1612 | INFO | 20/06/2019 09:19:25 PM] Key loaded into system!
[1612 | INFO | 20/06/2019 09:19:25 PM] Found jar directory!
[1612 | INFO | 20/06/2019 09:19:25 PM] Found server directory!
[1612 | INFO | 20/06/2019 09:19:25 PM] Found backup directory!
[1612 | INFO | 20/06/2019 09:19:25 PM] ----------
[1612 | INFO | 20/06/2019 09:19:25 PM] Initiating schedule worker..
[1612 | INFO | 20/06/2019 09:19:25 PM] Initiating file worker..
[1612 | INFO | 20/06/2019 09:19:25 PM] Initiating wrapper startup..
[1612 | INFO | 20/06/2019 09:19:25 PM] Binding socket to port 20000
[1620 | DEBUG | 20/06/2019 09:19:25 PM] Setting address to be reusable
[1612 | INFO | 20/06/2019 09:19:25 PM] Socket listening.
[1612 | INFO | 20/06/2019 09:19:42 PM] ----------
[1612 | INFO | 20/06/2019 09:19:42 PM] Loading server startup sequence..
[1612 | INFO | 20/06/2019 09:19:42 PM] Resetting server status..
[1612 | INFO | 20/06/2019 09:19:42 PM] Fetching server data..
[1612 | INFO | 20/06/2019 09:19:42 PM] Data fetched! Trying to start MineCraft Server..
[1612 | INFO | 20/06/2019 09:19:42 PM] Configuring wrapper for start of MineCraft Server..
[1612 | INFO | 20/06/2019 09:19:42 PM] Checking OS type..
[1612 | INFO | 20/06/2019 09:19:42 PM] Compiling startup command..
[1612 | INFO | 20/06/2019 09:19:42 PM] Compiled command: java -client -Duser.language=en -Xms1024M -Xmx16384M  -jar server.jar nogui --nojline
[1612 | INFO | 20/06/2019 09:19:42 PM] Continuing startup of MineCraft Server..
[1612 | INFO | 20/06/2019 09:19:42 PM] Updating global serverlist with server data..
[1620 | DEBUG | 20/06/2019 09:19:42 PM] Serverlist updated with {22b4c40b9275d91a8c3a38e139207ab4, <subprocess.Popen object at 0x867606c>, 10184}
[1612 | INFO | 20/06/2019 09:19:42 PM] Starting server daemon..
[1612 | INFO | 20/06/2019 09:19:42 PM] Server daemon started!
[1612 | INFO | 20/06/2019 09:19:42 PM] Set global server status to 'starting'..
[1612 | INFO | 20/06/2019 09:19:42 PM] MineCraft Server was successfully started!
[1612 | INFO | 21/06/2019 02:46:47 PM] *******************************
[1612 | INFO | 21/06/2019 02:46:47 PM] * New Wrapper Session Started *
[1612 | INFO | 21/06/2019 02:46:47 PM] *******************************
[1612 | INFO | 21/06/2019 02:46:47 PM] Checking if we should fix something before starting the wrapper..
[1612 | INFO | 21/06/2019 02:46:47 PM] Found configuration!
[1612 | INFO | 21/06/2019 02:46:47 PM] Found database!
[1612 | INFO | 21/06/2019 02:46:47 PM] Found wrapper key!
[1612 | INFO | 21/06/2019 02:46:47 PM] Trying to load key..
[1612 | INFO | 21/06/2019 02:46:47 PM] Key loaded into system!
[1612 | INFO | 21/06/2019 02:46:47 PM] Found jar directory!
[1612 | INFO | 21/06/2019 02:46:47 PM] Found server directory!
[1612 | INFO | 21/06/2019 02:46:47 PM] Found backup directory!
[1612 | INFO | 21/06/2019 02:46:47 PM] ----------
[1612 | INFO | 21/06/2019 02:46:48 PM] Initiating schedule worker..
[1612 | INFO | 21/06/2019 02:46:48 PM] Initiating file worker..
[1612 | INFO | 21/06/2019 02:46:48 PM] Initiating wrapper startup..
[1612 | INFO | 21/06/2019 02:46:48 PM] Binding socket to port 20000
[1620 | DEBUG | 21/06/2019 02:46:48 PM] Setting address to be reusable
[1612 | INFO | 21/06/2019 02:46:48 PM] Socket listening.
[1612 | INFO | 21/06/2019 02:47:17 PM] ----------
[1612 | INFO | 21/06/2019 02:47:17 PM] Loading server startup sequence..
[1612 | INFO | 21/06/2019 02:47:17 PM] Resetting server status..
[1612 | INFO | 21/06/2019 02:47:17 PM] Fetching server data..
[1612 | INFO | 21/06/2019 02:47:17 PM] Data fetched! Trying to start MineCraft Server..
[1612 | INFO | 21/06/2019 02:47:17 PM] Configuring wrapper for start of MineCraft Server..
[1612 | INFO | 21/06/2019 02:47:17 PM] Checking OS type..
[1612 | INFO | 21/06/2019 02:47:17 PM] Compiling startup command..
[1612 | INFO | 21/06/2019 02:47:17 PM] Compiled command: java -client -Duser.language=en -Xms1024M -Xmx16384M  -jar server.jar nogui --nojline
[1612 | INFO | 21/06/2019 02:47:17 PM] Continuing startup of MineCraft Server..
[1612 | INFO | 21/06/2019 02:47:17 PM] Updating global serverlist with server data..
[1620 | DEBUG | 21/06/2019 02:47:17 PM] Serverlist updated with {22b4c40b9275d91a8c3a38e139207ab4, <subprocess.Popen object at 0x899992c>, 30474}
[1612 | INFO | 21/06/2019 02:47:17 PM] Starting server daemon..
[1612 | INFO | 21/06/2019 02:47:17 PM] Server daemon started!
[1612 | INFO | 21/06/2019 02:47:17 PM] Set global server status to 'starting'..
[1612 | INFO | 21/06/2019 02:46:47 PM] Checking if we should fix something before starting the wrapper..
[1612 | INFO | 21/06/2019 02:46:47 PM] Found configuration!
[1612 | INFO | 21/06/2019 02:46:47 PM] Found database!
[1612 | INFO | 21/06/2019 02:46:47 PM] Found wrapper key!
[1612 | INFO | 21/06/2019 02:46:47 PM] Trying to load key..
[1612 | INFO | 21/06/2019 02:46:47 PM] Key loaded into system!
[1612 | INFO | 21/06/2019 02:46:47 PM] Found jar directory!
[1612 | INFO | 21/06/2019 02:46:47 PM] Found server directory!
[1612 | INFO | 21/06/2019 02:46:47 PM] Found backup directory!
[1612 | INFO | 21/06/2019 02:46:47 PM] ----------
[1612 | INFO | 21/06/2019 02:46:48 PM] Initiating schedule worker..
[1612 | INFO | 21/06/2019 02:46:48 PM] Initiating file worker..
[1612 | INFO | 21/06/2019 02:46:48 PM] Initiating wrapper startup..
[1612 | INFO | 21/06/2019 02:46:48 PM] Binding socket to port 20000
[1620 | DEBUG | 21/06/2019 02:46:48 PM] Setting address to be reusable
[1612 | INFO | 21/06/2019 02:46:48 PM] Socket listening.
[1612 | INFO | 21/06/2019 02:47:17 PM] ----------
[1612 | INFO | 21/06/2019 02:47:17 PM] Loading server startup sequence..
[1612 | INFO | 21/06/2019 02:47:17 PM] Resetting server status..
[1612 | INFO | 21/06/2019 02:47:17 PM] Fetching server data..
[1612 | INFO | 21/06/2019 02:47:17 PM] Data fetched! Trying to start MineCraft Server..
[1612 | INFO | 21/06/2019 02:47:17 PM] Configuring wrapper for start of MineCraft Server..
[1612 | INFO | 21/06/2019 02:47:17 PM] Checking OS type..
[1612 | INFO | 21/06/2019 02:47:17 PM] Compiling startup command..
[1612 | INFO | 21/06/2019 02:47:17 PM] Compiled command: java -client -Duser.language=en -Xms1024M -Xmx16384M  -jar server.jar nogui --nojline
[1612 | INFO | 21/06/2019 02:47:17 PM] Continuing startup of MineCraft Server..
[1612 | INFO | 21/06/2019 02:47:17 PM] Updating global serverlist with server data..
[1620 | DEBUG | 21/06/2019 02:47:17 PM] Serverlist updated with {22b4c40b9275d91a8c3a38e139207ab4, <subprocess.Popen object at 0x899992c>, 30474}
[1612 | INFO | 21/06/2019 02:47:17 PM] Starting server daemon..
[1612 | INFO | 21/06/2019 02:47:17 PM] Server daemon started!
[1612 | INFO | 21/06/2019 02:47:17 PM] Set global server status to 'starting'..
[1612 | INFO | 21/06/2019 02:47:17 PM] MineCraft Server was successfully started!
[1612 | INFO | 21/06/2019 02:56:36 PM] *******************************
[1612 | INFO | 21/06/2019 02:56:36 PM] * New Wrapper Session Started *
[1612 | INFO | 21/06/2019 02:56:36 PM] *******************************
[1612 | INFO | 21/06/2019 02:56:36 PM] Checking if we should fix something before starting the wrapper..
[1612 | INFO | 21/06/2019 02:56:36 PM] Found configuration!
[1612 | INFO | 21/06/2019 02:56:36 PM] Found database!
[1612 | INFO | 21/06/2019 02:56:36 PM] Found wrapper key!
[1612 | INFO | 21/06/2019 02:56:36 PM] Trying to load key..
[1612 | INFO | 21/06/2019 02:56:36 PM] Key loaded into system!
[1612 | INFO | 21/06/2019 02:56:36 PM] Found jar directory!
[1612 | INFO | 21/06/2019 02:56:36 PM] Found server directory!
[1612 | INFO | 21/06/2019 02:56:36 PM] Found backup directory!
[1612 | INFO | 21/06/2019 02:56:36 PM] ----------
[1612 | INFO | 21/06/2019 02:56:37 PM] Initiating schedule worker..
[1612 | INFO | 21/06/2019 02:56:37 PM] Initiating file worker..
[1612 | INFO | 21/06/2019 02:56:37 PM] Initiating wrapper startup..
[1612 | INFO | 21/06/2019 02:56:37 PM] Binding socket to port 20000
[1620 | DEBUG | 21/06/2019 02:56:37 PM] Setting address to be reusable
[1612 | INFO | 21/06/2019 02:56:37 PM] Socket listening.
[1612 | INFO | 21/06/2019 02:56:55 PM] ----------
[1612 | INFO | 21/06/2019 02:56:55 PM] Loading server startup sequence..
[1612 | INFO | 21/06/2019 02:56:55 PM] Resetting server status..
[1612 | INFO | 21/06/2019 02:56:55 PM] Fetching server data..
[1612 | INFO | 21/06/2019 02:56:55 PM] Data fetched! Trying to start MineCraft Server..
[1612 | INFO | 21/06/2019 02:56:55 PM] Configuring wrapper for start of MineCraft Server..
[1612 | INFO | 21/06/2019 02:56:55 PM] Checking OS type..
[1612 | INFO | 21/06/2019 02:56:55 PM] Compiling startup command..
[1612 | INFO | 21/06/2019 02:56:55 PM] Compiled command: java -client -Duser.language=en -Xms1024M -Xmx16384M  -jar server.jar nogui --nojline
[1612 | INFO | 21/06/2019 02:56:55 PM] Continuing startup of MineCraft Server..
[1612 | INFO | 21/06/2019 02:56:55 PM] Updating global serverlist with server data..
[1620 | DEBUG | 21/06/2019 02:56:55 PM] Serverlist updated with {22b4c40b9275d91a8c3a38e139207ab4, <subprocess.Popen object at 0x98f38cc>, 30532}
[1612 | INFO | 21/06/2019 02:56:55 PM] Starting server daemon..
[1612 | INFO | 21/06/2019 02:56:55 PM] Server daemon started!
[1612 | INFO | 21/06/2019 02:56:55 PM] Set global server status to 'starting'..
[1612 | INFO | 21/06/2019 02:56:55 PM] MineCraft Server was successfully started!
[1612 | INFO | 21/06/2019 03:05:04 PM] ----------
[1612 | INFO | 21/06/2019 03:05:04 PM] Adding a new task..
[1620 | DEBUG | 21/06/2019 03:05:04 PM] Connecting to database..
[1620 | DEBUG | 21/06/2019 03:05:04 PM] Inserting data..
[1612 | INFO | 21/06/2019 03:07:39 PM] ----------
[1612 | INFO | 21/06/2019 03:07:39 PM] Removing task..
[1612 | INFO | 22/06/2019 07:08:09 PM] *******************************
[1612 | INFO | 22/06/2019 07:08:09 PM] * New Wrapper Session Started *
[1612 | INFO | 22/06/2019 07:08:09 PM] *******************************
[1612 | INFO | 22/06/2019 07:08:09 PM] Checking if we should fix something before starting the wrapper..
[1612 | INFO | 22/06/2019 07:08:09 PM] Found configuration!
[1612 | INFO | 22/06/2019 07:08:09 PM] Found database!
[1612 | INFO | 22/06/2019 07:08:09 PM] Found wrapper key!
[1612 | INFO | 22/06/2019 07:08:09 PM] Trying to load key..
[1612 | INFO | 22/06/2019 07:08:09 PM] Key loaded into system!
[1612 | INFO | 22/06/2019 07:08:09 PM] Found jar directory!
[1612 | INFO | 22/06/2019 07:08:09 PM] Found server directory!
[1612 | INFO | 22/06/2019 07:08:09 PM] Found backup directory!
[1612 | INFO | 22/06/2019 07:08:09 PM] ----------
[1612 | INFO | 22/06/2019 07:08:09 PM] Initiating schedule worker..
[1612 | INFO | 22/06/2019 07:08:09 PM] Initiating file worker..
[1612 | INFO | 22/06/2019 07:08:09 PM] Initiating wrapper startup..
[1612 | INFO | 22/06/2019 07:08:09 PM] Binding socket to port 20000
[1620 | DEBUG | 22/06/2019 07:08:09 PM] Setting address to be reusable
[1612 | INFO | 22/06/2019 07:08:09 PM] Socket listening.
[1612 | INFO | 22/06/2019 07:08:18 PM] ----------
[1612 | INFO | 22/06/2019 07:08:18 PM] Loading server startup sequence..
[1612 | INFO | 22/06/2019 07:08:18 PM] Resetting server status..
[1612 | INFO | 22/06/2019 07:08:18 PM] Fetching server data..
[1612 | INFO | 22/06/2019 07:08:18 PM] Data fetched! Trying to start MineCraft Server..
[1612 | INFO | 22/06/2019 07:08:18 PM] Configuring wrapper for start of MineCraft Server..
[1612 | INFO | 22/06/2019 07:08:18 PM] Checking OS type..
[1612 | INFO | 22/06/2019 07:08:18 PM] Compiling startup command..
[1612 | INFO | 22/06/2019 07:08:18 PM] Compiled command: java -client -Duser.language=en -Xms1024M -Xmx16384M  -jar server.jar nogui --nojline
[1612 | INFO | 22/06/2019 07:08:18 PM] Continuing startup of MineCraft Server..
[1612 | INFO | 22/06/2019 07:08:18 PM] Updating global serverlist with server data..

**Comment by TunkDesign** [comment]
Strange...! The log tells me that the server was started. 

What happens if you run the server the normal way? Does it stay online, or die as in your issue?

**Comment by SmuleChez** [comment]
With out the wrapper the server runs perfectly fine, it will run for hours without any issue

**Comment by TunkDesign** [comment]
Any mods/plugins? Regular minecraft? I'd like to do some testing :)

**Comment by SmuleChez** [comment]
It is completely vanilla minecraft server, I have no plugins or mods, Thanks!
